e2e/wdio/core/tests/select.e2e-spec.ts

```diff
@@ -0,0 +1,398 @@
+import { SelectPo } from '../pages/select.po';
+import {
+    addIsActiveClass,
+    checkElementScreenshot,
+    click,
+    focusElement,
+    getAttributeByName,
+    getElementArrayLength,
+    getImageTagBrowserPlatform,
+    getText, isElementClickable,
+    isElementDisplayed,
+    mouseHoverElement,
+    refreshPage,
+    saveElementScreenshot,
+    waitForElDisplayed,
+    waitForInvisibilityOf
+} from '../../driver/wdio';
+import {
+    addRemoveSelectOptions,
+    customControlSelectOptions,
+    extendedSelectOptions,
+    maxHeightSelectOptions,
+    mobileSelectOptions,
+    optionTag,
+    programmaticSelectOptions,
+    selectModesCompactSelectOptions,
+    selectModesDefaultSelectOptions,
+    semanticModesErrorSelectOptions,
+    semanticModesInformationSelectOptions,
+    semanticModesSuccesSelectOptions,
+    semanticModesWarningSelectOptions
+} from '../fixtures/testData/select-tags';
+
+describe('Select component:', function() {
+    beforeEach(() => {
+        refreshPage();
+    });
+
+    const selectPage = new SelectPo();
+    const {
+        selectModesExample,
+        selectSemanticStatesExample,
+        customControlExample,
+        extendetOptionsExample,
+        mobileModeExample,
+        maxHeightExample,
+        addRemoveOptionExample,
+        programmaticControlExample,
+        overlayContainer,
+        buttons,
+        option,
+        listBox,
+        options,
+        displayedText
+    } = selectPage;
+
+
+    beforeAll(() => {
+        selectPage.open();
+    }, 1);
+
+    describe('Select modes', function() {
+        it('should be able to select the option for default select', () => {
+            const textBefore = getText(selectModesExample + displayedText, 0);
+            click(selectModesExample + buttons, 0);
+            waitForElDisplayed(option, 4);
+            click(options(2));
+            const textAfter = getText(selectModesExample + displayedText, 0);
```

Comment:
is index 0 needed here? same for the couple lines above

Reply:
Fixed